.codecov.yml

```diff
@@ -1,2 +1,37 @@
 codecov:
   branch: develop
+
+comment: off
+
+coverage:
+  status:
+    patch:
+      default: off
+      source:
+        target: 80%
+        flags: source
+      backend:
+        target: 80%
+        flags: backend
+    project:
+      default: off
+      source:
+        flags: source
+      backend:
+        flags: backend
+
+flags:
+  source:
+    paths:
+      - src/
+  backend:
+    paths:
+      - mod/
+      - mods/
```

Comment:
Does this mean to include mods in the backend path or exclude it? mods is used for example files, don't think that needs to be covered.

Reply:
mod will be included
=> I'll remove it

Reply:
`mod` is correct, but `mods` just holds examples.